nodejs/Dockerfile-kali

```diff
@@ -0,0 +1,21 @@
+# A docker image for the func container.
+
+# debian variant is the official standard node image (larger than the alpine image)
+FROM kalilinux/kali-rolling
+
+RUN apt update && apt install -y curl nodejs
```

Comment:
Should we pin this to a specific version of NodeJS otherwise it is open for future changes and potential issues. In case of other envs, we use a specific version of Node Docker image hence it works fine: https://github.com/fission/environments/blob/master/nodejs/Dockerfile